### PR TITLE
some cleanup fixes

### PR DESCRIPTION
**Pull request by andymckay**
- tidy up diagrams to reflect what got built
- remove API and point to solitude which has it all
- add in payments-example

### CODE REVIEW

File: docs/design/overview.rst [hunk 1]
```diff
@@ -101,9 +101,9 @@ Added to the site selling the product. The library allows the selling site to
 easily bring up the iframe for completing the payment flow and then processing
 the completed flow.
 
-* *UX*: none
+* *UX*: :ref:`buy flow <purchase-label>`
 * *Documentation*: ...
-* *Repository*: ...
+* *Repository*: https://github.com/mozilla/payments-clients
```
Comment (kumar303) [inline docs/design/overview.rst]: I think you meant https://github.com/mozilla/payments-client

File: docs/design/overview.rst [hunk 2]
```diff
@@ -129,6 +129,15 @@ Firefox Accounts profile page.
 * *Uptime requirements*: high, this doesn't stop purchasing work but prevents
   later management.
 
+Example site
+++++++++++++
+
+An example site that shows how
+
+* *Repository*: https://github.com/mozilla/payments-example
+* *Uptime requirements*: none, its an example.
```
Comment (kumar303) [inline docs/design/overview.rst:138]: ha

**Comment by kumar303** [comment]
In the [subscription flow](https://raw.githubusercontent.com/andymckay/payments/doc-clean-up/docs/design/start-subscription.png) the buyer can choose an existing payment method, not sure if we need to add that to the flow or not.

**Comment by kumar303** [comment]
There's a box for checking if the subscription exists or not but actually I think we always create a new subscription as part of this flow.

**Comment by kumar303** [comment]
In other words, there is no difference between creating a subscription and charging a subscription, right?

**Comment by kumar303** [comment]
or by "creating a subscription" do you mean create a product?

**Comment by kumar303** [comment]
r+wc

**Comment by andymckay** [comment]
Braintree allows you to create a subscription but not charge for it straight away. You could have a 10 day trial before we charge. That's currently done by this: https://github.com/mozilla/solitude/blob/master/lib/brains/forms.py#L85, but if we didn't have that, it would be two steps.

File: docs/design/overview.rst (hunk 1 above)
Comment (andymckay) [inline docs/design/overview.rst]: Danke.

**Pull request merged**